rust/parser/test/mod.rs

```diff
@@ -21,8 +21,9 @@
  */
 
```

Comment:
Most changes here are replacing the top-level builders with the corresponding macros and reformatting of uncommented tests.